composer.json

```diff
@@ -85,14 +85,14 @@
         "laravel/passport": "^8.2",
         "laravel/telescope": "^3.3",
         "laravel/tinker": "^2.0",
+        "laravel/ui": "^2.0",
         "laravelcollective/html": "^6.0",
         "league/csv": "^8.0",
         "maatwebsite/excel": "~3.1.17",
         "malahierba-lab/public-id": "dev-master",
         "planetteamspeak/ts3-php-framework": "1.1.33",
         "predis/predis": "^1.1",
         "pusher/pusher-php-server": "~4.0",
-        "sixlive/nova-text-copy-field": "^1.5",
```

Comment:
removing this will break the feedback resource

Reply:
It is in `composer-nova.json` which is merged to the main `composer.json` when required. It has been referenced in both files for some reason, this only removes the duplicate.